zyte_api/_retry.py

```diff
@@ -55,8 +55,61 @@ def _is_temporary_download_error(exc: BaseException) -> bool:
 
 
 class RetryFactory:
-    """
-    Build custom retry configuration
+    """Factory class that builds the :class:`tenacity.AsyncRetrying` object
+    that defines the :ref:`default retry policy <default-retry-policy>`.
+
+    To create a custom retry policy, you can subclass this factory class,
+    modify it as needed, and then call :meth:`build` on your subclass to get
+    the corresponding :class:`tenacity.AsyncRetrying` object.
+
+    For example, to increase the maximum number of attempts for :ref:`temporary
```

Comment:
From this point forward it is a copy of https://scrapy-zyte-api.readthedocs.io/en/latest/usage/retry.html. I’m thinking it may make sense to keep useful retry policy examples in python-zyte-api docs and point to them from scrapy-zyte-api docs (where we may leave 1 entire example, I’m still not sure).

Whether we keep the examples in this docstring, in a separate doc page, or somewhere else, no strong opinion.

Reply:
It looks fine to me.